doc/en/user/source/community/metadata/configuration.rst

```diff
@@ -0,0 +1,202 @@
+      typename: data-identification-date
+      occurrence: REPEAT            
+  types:    
+     - typename: data-identification-date
+       attributes:
+        - key: date
+          fieldType: DATE
+        - key: date-type
+          fieldType: DROPDOWN
+          values:
+            - creation
+            - publication
+            - revision  
+
+This configuration results in the following GUI:
+
+.. figure:: images/basic-gui.png
+
+
+
+There are 2 main parts in the `yaml <https://yaml.org/>`__:
+
+    - **attributes:** a list of GUI components that will be rendered in the tab. They can be a basic type or a complex type, a complex type is a collection of basic types.
+    - **types:** a list that defines the fields in each complex type.
+
+:ref:`community_metadata_uiconfiguration` gives an overview of all supported types and advanced features.
+
+
+Import from Geonetwork mapping
+------------------------------
+The :guilabel:`Import from Geonetwork` option allows the user to import existing metadata from `GeoNetwork <https://geonetwork-opensource.org//>`_.
+Two confurations are needed for the import to work:
+
+    - **geonetworks:** configure a list geonetwork endpoints
+    - **geonetworkmapping:** define the mapping between the geonetwork fields and the fields configured in the metadata module.
+
+The configuration can be added to the same `yaml <https://yaml.org/>`__ file as the UI configuration or it can be put in a separate file.
+
+Enpoint configuration
+^^^^^^^^^^^^^^^^^^^^^
+The example will configure 2 endpoints. 
+
+.. code:: YAML
+
+    geonetworks:
+        - name: Geonetwork DOV production
+          url: https://www.dov.vlaanderen.be/geonetwork/
+        - name: Geonetwork test
+          url: https://geonetwork-opensource.org/test
+
+
+
+================  ========  ============================
+Key               Required  Description
+================  ========  ============================
+**name**           yes       The name fof the geonetwork endpoint that will be shown in the dropdown
+**url**            yes       The url of the geonetwork
+================  ========  ============================
+
+Mapping configuration
+^^^^^^^^^^^^^^^^^^^^^
+The example will map one field (UUID) from the geonetwork xml to UI.
+
+.. code:: YAML    
+    
+    geonetworkmapping:
+        -  geoserver: metadata-identifier
+           geonetwork: //gmd:fileIdentifier/gco:CharacterString/text()
+
+================  ========  ============================
+Key               Required  Description
+================  ========  ============================
+**geoserver**      yes      the key for the attributes in geoserver
+**geonetwork**     yes      The `xpath <https://developer.mozilla.org/en-US/docs/Web/XPath>`__ expression pointing to the content from the geonetwork metadata xml file.
+================  ========  ============================
+
+Native attribute mapping
+------------------------
+.. warning:: TODO
+
```

Comment:
Alles hieronder moet in een aparte tutorial, er mag hier een link naartoe geplaatst worden (maar toch zeker ook vanaf de index)